src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-flow-modal/rgw-multisite-sync-flow-modal.component.ts

```diff
@@ -162,8 +153,21 @@ export class RgwMultisiteSyncFlowModalComponent implements OnInit {
       this.currentFormGroupContext.setErrors({ cdSubmitButton: true });
       return;
     }
+
+    const zones: any = { added: [], removed: [] };
```

Comment:
Add proper types here for type safety
```suggestion
    const zones: ZoneType = { added: [], removed: [] };
```